CHANGELOG.md

```diff
@@ -28,7 +72,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Fix that the screen reader is not able to read options of the dropdown lists as user navigates. [#961](https://github.com/rokwire/events-manager/issues/961)
 - Fix the issue that hours and seconds will be displayed for all day events in calendar events. [#958](https://github.com/rokwire/events-manager/issues/958)
 - Set the empty location description if the user event is physical event and no location provided. [#981](https://github.com/rokwire/events-manager/issues/981)
+
 ### Changed
+- Change font size and color for text.[#975](https://github.com/rokwire/events-manager/issues/975)
```

Comment:
Similar thing with this entry too. I think it needs to be moved to the Changed section under 3.5.0.

Reply:
fixed